exporters/metric/cortex/example/go.mod

```diff
@@ -6,6 +6,6 @@ require (
 	go.opentelemetry.io/contrib/exporters/metric/cortex v0.10.1
 	go.opentelemetry.io/contrib/exporters/metric/cortex/utils v0.10.1
 	go.opentelemetry.io/otel v0.10.0
-	go.opentelemetry.io/otel/exporters/metric/prometheus v0.10.0
 	go.opentelemetry.io/otel/sdk v0.10.0
+	gopkg.in/yaml.v2 v2.2.5 // indirect
```

Comment:
Where is this indirect dependency coming from all of a sudden? It seems no other dependencies have changed besides removal of the `prometheus` package with this PR. Does running `go mod tidy` remove this?

Reply:
This is in the example project which uses the config utils which rely on yaml. I think that's where it comes from. I ran `make precommit` which recursively runs tests and go mod tidy (and some other go utils) throughout the entire repo so I think it's required.